### ISSUE TITLE
Error in hive sql command

### ISSUE THREAD

**Issue by ssuvorov-fls** [OHDSI/WebAPI#1749]
Error occures during generating cohort

```
org.springframework.jdbc.BadSqlGrammarException: StatementCallback; 
bad SQL grammar [DELETE FROM omop_orc_results_280.cohort_inclusion WHERE cohort_definition_id = 77081;]; 
nested exception is org.apache.hive.service.cli.HiveSQLException: Error while compiling statement: 
FAILED: ParseException line 1:84 cannot recognize input near '77081' ';' '<EOF>' in constant
```

**Comment by chrisknoll** [comment]
I believe this is an example where we need to translate and `SplitSql` the delete command.

Possibly a dupe of #1730.

**Comment by ssuvorov-fls** [comment]
closed via #1730
